docs/portal/transports/add-transport.md

```diff
@@ -11,6 +11,7 @@ The <strong> Transports </strong> page can be reached simply by clicking "Transp
 | Transport Name       | Enter a name that helps you identify this transport, if you leave it empty, the HDT portal will generate a default name in the format of "Target Domain:Target Port" |
```

Comment:
Please consider the following: Enter a name to identify this transport. If you omit this parameter, the HDT portal generates a default name in the format of "Target Domain:Target Port".

Reply:
Will update it.